_training/curriculum.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "Towards a HEP Software Training curriculum"
+layout: plain
+---
+
+<style type="text/css">
```

Comment:
Probably not the best style to just paste this here, but seems to work well.

Reply:
I don't think it's so important!